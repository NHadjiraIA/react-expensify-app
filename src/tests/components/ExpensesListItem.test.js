import ExpenseListItem from '../../components/ExpenseListItem'
import React from 'react'
import { shallow } from 'enzyme'
import expenses from '../fixtures/expenses'


test ('should render expensesListItem  with expenses  correctly', () =>{
    const wrapper = shallow(<ExpenseListItem  {...expenses[0]}/>);
    expect(wrapper).toMatchSnapshot();
})