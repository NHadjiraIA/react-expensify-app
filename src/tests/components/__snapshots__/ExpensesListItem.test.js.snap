// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensesListItem  with expenses  correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h1>
       
      Gaz
    </h1>
  </Link>
  <p>
     
    123
     - 
    0
  </p>
</div>
`;
